hpobench/container/client_abstract_benchmark.py

```diff
@@ -64,79 +64,82 @@ class AbstractBenchmarkClient(metaclass=abc.ABCMeta):
     """
     def __init__(self, benchmark_name: str, container_name: str, container_source: Optional[str] = None,
                  container_tag: str = 'latest', env_str: Optional[str] = '', bind_str: Optional[str] = '',
-                 gpu: Optional[bool] = False, rng: Union[np.random.RandomState, int, None] = None, **kwargs):
+                 gpu: Optional[bool] = False, rng: Union[np.random.RandomState, int, None] = None,
+                 socket_id=None, **kwargs):
```

Comment:
Could you copy/adapt the docstring from previous _setup() and also explain what socket_id is or should be

Reply:
done